torchvision/transforms/v2/_auto_augment.py

```diff
@@ -15,6 +15,59 @@
 from .utils import check_type, is_simple_tensor
 
 
+def solarize_add(
```

Comment:
Let's make this method private : `_solarize_add`
I'm not very sure about Pillow implementation and if we could not use other pillow methods like blend here...
Ideally, it would be better to introduce this method into functional part and test explicitly...